redfish-core/lib/led.hpp

```diff
@@ -238,25 +239,56 @@ inline void
                                nlohmann::json& jsonInput)
 {
     BMCWEB_LOG_DEBUG << "Get LocationIndicatorActive";
-
     nlohmann::json& jsonIn = jsonInput["LocationIndicatorActive"];
-    dbus::utility::getAssociationEndPoints(
-        objPath + "/identifying",
-        [aResp, &jsonIn](const boost::system::error_code& ec,
-                         const dbus::utility::MapperEndPoints& endpoints) {
+
+    sdbusplus::asio::getProperty<
+        std::vector<std::tuple<std::string, std::string, std::string>>>(
+        *crow::connections::systemBus, "xyz.openbmc_project.Inventory.Manager",
+        objPath, "xyz.openbmc_project.Association.Definitions", "Associations",
+        [aResp, objPath,
+         &jsonIn](const boost::system::error_code ec,
+                  std::vector<std::tuple<std::string, std::string, std::string>>
+                      associations) {
```

Comment:
I think this is better to be written like

```
 dbus::utility::getAssociationList(
         "xyz.openbmc_project.Inventory.Manager", objPath,
        [aResp, objPath, & jsonIn](
            const boost::system::error_code& ec,
            const dbus::utility::AssociationList& associations) {
        if (ec)
       {
       }
```
Here
`std::vector<std::tuple<std::string, std::string, std::string>` == `dbus::utility::AssociationList``

Reply:
ok. Thanks for pointing to that.